config.go

```diff
@@ -49,8 +49,11 @@ type config struct {
 	// PrimaryNode is the pubkey of the primary node in primary-gateway setups.
 	PrimaryNode string `long:"primarynode" description:"Public key of the primary node in a primary-gateway setup"`
 
-	// DisableGraph disables collection of graph metrics
+	// DisableGraph disables collection of graph metrics.
 	DisableGraph bool `long:"disablegraph" description:"Do not collect graph metrics"`
+	
```

Comment:
There's a stray tab character on this line. You need to run `go fmt` on the code.